dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/Ds3PutJob.java

```diff
@@ -97,7 +98,16 @@ public Ds3PutJob(final Ds3Client client,
         this.ds3Client = client;
         this.resourceBundle = resourceBundle;
         this.files = files.stream()
-                .map(p -> new Pair<>(p.getKey().replace(delimiter, BP_DELIMITER), p.getValue()))
+                .map(p -> {
+                    try {
+                        return new Pair<>(p.getKey().replace(delimiter, BP_DELIMITER), FileUtils.resolveForSymbolic(p.getValue()));
+                    } catch (final IOException e) {
+                        loggingService.logMessage("Could not readd from filesystem, aborting put job", LogType.ERROR);
```

Comment:
This won't abort the putjob.  It will just cause that entry to not show up.  We should update this error message to state that, and say what file we had issues with.